scripts/set-more-info-link.py

```diff
@@ -6,50 +6,49 @@
 import sys
 
 labels = {
-    'en': 'More information:',
-    'bs': 'Više informacija:',
-    'da': 'Mere information:',
-    'de': 'Weitere Informationen:',
-    'es': 'Más información:',
-    'fa': 'اطلاعات بیشتر:',
-    'fr': 'Plus d\'informations\xa0:',
-    'sh': 'Više informacija:',
-    'hi': 'अधिक जानकारी:',
-    'id': 'Informasi lebih lanjut:',
-    'it': 'Maggiori informazioni:',
-    'ja': '詳しくはこちら:',
-    'ko': '더 많은 정보:',
-    'ml': 'കൂടുതൽ വിവരങ്ങൾ:',
-    'nl': 'Meer informatie:',
-    'no': 'Mer informasjon:',
-    'pl': 'Więcej informacji:',
-    'pt_BR': 'Mais informações:',
-    'pt_PT': 'Mais informações:',
-    'ru': 'Больше информации:',
-    'sv': 'Mer information:',
-    'ta': 'மேலும் தகவல்:',
-    'th': 'ดูเพิ่มเติม:',
-    'tr': 'Daha fazla bilgi için:',
-    'zh_TW': '更多資訊：',
-    'zh': '更多信息：',
+    "en": "More information:",
+    "bs": "Više informacija:",
+    "da": "Mere information:",
+    "de": "Weitere Informationen:",
+    "es": "Más información:",
+    "fa": "اطلاعات بیشتر:",
+    "fr": "Plus d'informations\xa0:",
+    "sh": "Više informacija:",
+    "hi": "अधिक जानकारी:",
+    "id": "Informasi lebih lanjut:",
+    "it": "Maggiori informazioni:",
+    "ja": "詳しくはこちら:",
+    "ko": "더 많은 정보:",
+    "ml": "കൂടുതൽ വിവരങ്ങൾ:",
+    "nl": "Meer informatie:",
+    "no": "Mer informasjon:",
+    "pl": "Więcej informacji:",
+    "pt_BR": "Mais informações:",
+    "pt_PT": "Mais informações:",
+    "ru": "Больше информации:",
+    "sv": "Mer information:",
+    "ta": "மேலும் தகவல்:",
+    "th": "ดูเพิ่มเติม:",
+    "tr": "Daha fazla bilgi için:",
+    "zh_TW": "更多資訊：",
+    "zh": "更多信息：",
 }
 
-IGNORE_FILES = (
-    '.DS_Store',
-)
+IGNORE_FILES = (".DS_Store",)
 
 
 def get_tldr_root():
     # if this script is running from tldr/scripts, the parent's parent is the root
     f = os.path.normpath(__file__)
-    if f.endswith('tldr/scripts/set-more-info-link.py'):
+    if f.endswith("tldr/scripts/set-more-info-link.py"):
         return os.path.dirname(os.path.dirname(f))
 
-    if 'TLDR_ROOT' in os.environ:
-        return os.environ['TLDR_ROOT']
+    if "TLDR_ROOT" in os.environ:
+        return os.environ["TLDR_ROOT"]
     else:
         print(
-            '\x1b[31mPlease set TLDR_ROOT to the location of a clone of https://github.com/tldr-pages/tldr.')
+            "\x1b[31mPlease set TLDR_ROOT to the location of a clone of https://github.com/tldr-pages/tldr."
+        )
```

Comment:
Why don't we just put this in a single line?

Reply:
> Why don't we just put this in a single line?

`PEP8` (the standard for formatting), kinda push those kind of layouts especially with the line lenght limit (which is 72 char)